frontend/src/features/projects/spl/forms/CloseOutFinancialSummaryForm.tsx

```diff
@@ -28,7 +28,7 @@ const CloseOutFinancialSummaryForm = (props: CloseOutFinancialSummaryFormProps)
               formikProps={formikProps}
               disabled={props.isReadOnly}
               outerClassName="col-md-8"
-              field="salesProceeds"
+              field="market"
```

Comment:
Sales Proceeds is the same as Current Market Value, as that is how much was paid.